app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -123,6 +123,22 @@ examples:
           value: "green"
         - label: "Blue"
           value: "blue"
+  with_custom_size_for_page_header_and_caption:
+    description: |
+      If a custom heading size is passed in, the page heading and caption size will be changed to set size while still retaining a heading level.
+    data:
+      name: "favourite_skittle"
+      heading: "Choose your favourite skittles"
+      heading_caption: "Question 3 of 9"
+      heading_size: l
+      is_page_heading: true
```

Comment:
I think having the `is_page_heading` option here detracts from the point of the example. The interaction of `is_page_heading` and `heading_size` is a little complicated, I'd suggest it be explained in the description instead.